sdk/identity/azure-identity/src/main/java/com/azure/identity/InteractiveBrowserCredentialBuilder.java

```diff
@@ -66,6 +71,21 @@ InteractiveBrowserCredentialBuilder authenticationRecord(AuthenticationRecord au
         return this;
     }
 
+
+    /**
+     * Sets the Redirect URL where STS will callback the application with the security code. It is required if a custom
+     * client id is specified via {@link InteractiveBrowserCredentialBuilder#clientId(String)} and must match the
+     * redirect URL specified during the application registration.
+     *
+     * @param redirectURL the redirect URL to listen on and receive security code.
+     *
+     * @return An updated instance of this builder with the configured redirect URL.
+     */
+    public InteractiveBrowserCredentialBuilder redirectUrl(String redirectURL) {
+        this.redirectURL = redirectURL;
```

Comment:
Can `redirectUrl` be null? Should we have a null check here?

Reply:
if its null, we localhost with an arbitrary valid port as default.